core/src/main/scala/org/apache/spark/rdd/ReliableCheckpointRDD.scala

```diff
@@ -100,10 +107,52 @@ private[spark] object ReliableCheckpointRDD extends Logging {
     "part-%05d".format(partitionIndex)
   }
 
+  private def checkpointPartitionerFileName(): String = {
+    "_partitioner"
+  }
+
   /**
-   * Write this partition's values to a checkpoint file.
+   * Write RDD to checkpoint files and return a ReliableCheckpointRDD representing the RDD.
    */
-  def writeCheckpointFile[T: ClassTag](
+  def createCheckpointedRDD[T: ClassTag](
```

Comment:
`ReliableCheckpointRDD.create` looks like "an *RDD* instance is created in memory", but as I understand the core part of this method is writing *checkpoint* data to file system. I would vote for `createCheckpoint`